C/jets.c

```diff
@@ -80,6 +93,208 @@ bool sha_256_block(frameItem* dst, frameItem src, const txEnv* env) {
   return true;
 }
 
+/* Read bytes from a Simplicty bufffer of type (TWO^8)^<2^(n+1) into 'buf'.
```

Comment:
In c56cae90402f1fc04084957df690bf4f4784d94f:

Two typos: `Simplicty` -> `Simplicity` and `<2^(n+1)` I think think should be `(2^(n+1))`.

These are both repeated below.

Reply:
Oh, a third -- `bufffer` to `buffer` :).

Reply:
I've fixed the typos.  I've added some comments documenting the `X^<n` notation.